cmd/generate.go

```diff
@@ -81,6 +82,40 @@ func genProcessCluster(cmd *cobra.Command, clusterName string, p *ants.Pool) {
 		}
 	}
 
+	// list of components in cluster
+	for c, _ := range clusterComponents {
+		var clComps []string
+		clComps = append(clComps, c)
```

Comment:
wont `clComps == clusterComponents` always be true?

Reply:
due to this - https://github.com/apptio/kr8/blob/master/cmd/generate.go#L42
I did above to turn this into slice, or it won't compile